### PR TITLE
Add animations on Gallery page. Add interactive Projects page. Add Welcome page with hero banner.

### PR DESCRIPTION
**Pull request by saraholiviawu**
No hover / click:
Screenshot 2020-05-28 at 10 02 18 PM - Display 2
Hover (button color changes to a lighter shade):
Screenshot 2020-05-28 at 10 02 22 PM - Display 2
Click (button moves downward in the y-direction):
Screenshot 2020-05-28 at 10 02 25 PM - Display 2

### CODE REVIEW

**Review by saraholiviawu — COMMENTED** [review]
Initial Gallery screen:
Screenshot 2020-05-28 at 10 32 06 PM - Display 2

Click on button:
Screenshot 2020-05-28 at 10 31 11 PM - Display 2

Click on button a second time:
Screenshot 2020-05-28 at 10 30 56 PM - Display 2

**Review by saraholiviawu — COMMENTED** [review]
No hover:
Screenshot 2020-05-28 at 11 15 45 PM - Display 2
Hover:
Screenshot 2020-05-28 at 11 15 46 PM - Display 2

**Review by saraholiviawu — COMMENTED** [review]
Screenshot 2020-05-29 at 12 26 47 AM - Display 2

**Review by saraholiviawu — COMMENTED** [review]
No hover:
Screenshot 2020-05-29 at 4 31 10 AM - Display 2

Hover Demonstration:
Screenshot 2020-05-29 at 4 31 23 AM - Display 2

Screenshot 2020-05-29 at 4 31 27 AM - Display 2

**Review by saraholiviawu — COMMENTED** [review]
The two icons are created using Adobe Illustrator; no need to reference a source.

**Review by saraholiviawu — COMMENTED** [review]
Screenshot 2020-05-29 at 8 05 59 AM - Display 2

**Review by saraholiviawu — COMMENTED** [review]
No hover:
Screenshot 2020-05-29 at 8 10 13 AM - Display 2

Hover:
Screenshot 2020-05-29 at 8 12 08 AM - Display 2

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <div class="navbar-link">
+            <a href="blog.html">BLOG</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="content">
+        <div class="two-column">
+            <div class="profile-image">
+                <img src="/images/profile.jpg" alt="Sarah's Profile Picture">
```
Comment (peter14f) [inline portfolio/src/main/webapp/about.html:42]: Seeing two whitespaces and four whitespaces being used in the same file. Not sure if the editor you're using can auto fix that to make it consistent.

File: portfolio/src/main/webapp/gallery.html [hunk 2]
```diff
@@ -41,8 +41,10 @@
         <div class="gal-cover-container">
           <h1>PHOTOGRAPHY</h1>
           <h2>Join me on my travels.</h2>
-         <button type="button" onclick="randomizeImage()" class="button">Venture</button>
+          <button type="button" onclick="randomizeImage()" class="button">VENTURE</button>
+          <p id="random-image-caption"></p>          
           <div id="random-image-container"></div>
+
       </div>
```
Comment (peter14f) [inline portfolio/src/main/webapp/gallery.html:48]: Fix the indentation so that it aligns with its opening tag.

**Review by peter14f — APPROVED** [review]

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (saraholiviawu) [reply portfolio/src/main/webapp/about.html]: Fixed

**Pull request merged**